pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -699,6 +694,27 @@ func TestEventListenerValidate_error(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "Specify TriggerTemplate along with TriggerRef",
+		el: &v1alpha1.EventListener{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "name",
+				Namespace: "namespace",
+			},
+			Spec: v1alpha1.EventListenerSpec{
+				Triggers: []v1alpha1.EventListenerTrigger{{
+					Template: &v1alpha1.EventListenerTemplate{
+						Ref: ptr.String("tt"),
+					},
+					TriggerRef: "triggerref",
+				}},
+				Resources: v1alpha1.Resources{
```

Comment:
is this field required for this test?

Reply:
Its not required 
Removed from testcase